doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -31,6 +31,9 @@ The following sections provide detailed lists of changes by component.
 Application development
 =======================
 
+* Implemented new configuration scheme for build types in application development and updated several applications accordingly.
+  The new scheme is based on Zephyr and uses application-specific :file:`prj.conf` files instead of build types selected through CMake build type variable (``CMAKE_BUILD_TYPE``).
```

Comment:
```suggestion
  The new scheme is based on Zephyr and uses application-specific :file:`prj.conf` files instead of build types selected through the CMake build type variable (``CMAKE_BUILD_TYPE``).
```